content/en/docs/tasks/security/rbac-groups/index.md

```diff
@@ -26,17 +26,17 @@ To fulfill this prerequisite you can follow the Kubernetes
 
 ## Setup the required namespace and services
```

Comment:
As the policy has been changed from service-oriented to workload-oriented, should "Setup the required namespace and services" be updated to "Setup the required namespace and workloads"? Similarly for the services mentioned in this section?